src/renderer/atlas/wic.h

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include <wincodec.h>
+
+inline void SaveTextureToPNG(ID3D11DeviceContext* deviceContext, ID3D11Resource* source, double dpi, const wchar_t* fileName)

```

Comment:
This function is only used in debug builds and it's an extract from https://github.com/microsoft/DirectXTK.

Reply:
About `GUID_ContainerFormatGif` hard. 😎